src/content/interop/js-interop/usage.md

```diff
@@ -507,6 +507,8 @@ Utility functions include:
   as the extra type-checking might be expensive.
 - [`importModule`][], which allows you to
   import modules dynamically as a `JSObject`.
+- [`isA`][], which allows you to check if a JS-interop value is
+  an instance of the JS type specified as the type argument.
```

Comment:
nit: Perhaps `by the type argument` instead?

Reply:
```suggestion
  an instance of the JS type specified by the type argument.
```